src/components/study-container.js

```diff
@@ -351,6 +351,10 @@ export function StudyContainer({ view, onChangeTab }) {
         // Note: studyUuid and dispatch don't change
     }, [studyUuid, dispatch, snackError, snackWarning]);
 
+    function limitChar(str) {
```

Comment:
Would be better to define it out of the component function, as it avoids a new parsing of it at each call of the component function.

Reply:
it's fixed